docs/fr/docs/advanced/additional-status-codes.md

```diff
@@ -0,0 +1,37 @@
+# Codes de statut supplémentaires
+
+Par défaut, **FastAPI** renverra les réponses à l'aide d'une structure de données `JSONResponse`, en plaçant la réponse de votre  *chemin d'accès* à l'intérieur de cette `JSONResponse`.
+
+Il utilisera le code d'état par défaut ou celui que vous avez défini dans votre *chemin d'accès*.
```

Comment:
I would use _statut_ instead of _état_

Reply:
@rjNemo : Wikipedia parle de "codes HTTP" (sans parler de _statut_ ou _d'état_) : [Liste des codes HTTP](https://fr.wikipedia.org/wiki/Liste_des_codes_HTTP)

Qu'en dis-tu ?

Reply:
Encore mieux!